### ISSUE TITLE
Add SubjectAttribute instances for animal research

### ISSUE THREAD

**Issue by ree-gupta** [openMetadataInitiative/openMINDS_instances#404]
In NHP and other animal research, subjects often have specific states related to their experimental history and motivational protocols that are important to document.

## Proposed SubjectAttribute Terms

| Term | Name | Definition |
|------|------|------------|
| `naive` | naive | A subject with no prior experimental history |
| `preTrained` | pre-trained | A subject that has received prior behavioral training |
| `foodRestricted` | food restricted | A subject under a controlled food restriction protocol |
| `waterRestricted` | water restricted | A subject under a controlled water restriction protocol |

## Question

Is `naive` distinct enough from `untreated` and `control` to warrant a separate term? In my understanding, `untreated` refers to no state-altering treatment, while `naive` specifically means no prior experimental involvement.

**Comment by lzehl** [comment]
@tgbugs what is your inside/opinion on naive? -> naive is fine; but careful with definition
@ree-gupta do you summarize the use cases where you like to have this?

pre-trained should be the term name as well (not preTrained, keep the -)
add over-trained as well (if not already there; double check spelling)

**Comment by tgbugs** [comment]
suggest `naive` -> `no-experimental-history` and then add `untrained-on-task` and `trained-on-task` and `trained-on-task` probably means `over-trained`. `untrained-on-task` means `naive-in-context` but doesn't say anything about potential prior exposure to the paradigm or to other similar tasks where there might be transfer, e.g. same task but change hand, or know task but only one part of the high dimensional stimulus is changed. If we use `pre-trained` it needs to be formulated very carefully to imply that the subject has been exposed to a _very_ similar task, or tasks with overlapping features, e.g. the reward design is similar, or the motor output is similar, or the sensory feed is similar, the the stimulus is similar, etc. `trained-on-similar-task` maybe is what `pre-trained` means, or `trained-on-task-component`.

**Comment by tgbugs** [comment]
reminder these are really `SubjectStateAttributes`.

**Comment by tgbugs** [comment]
as is right now this list would grow to cover essentially all boolean variables/states that could be associated with a subject, e.g. `stroke?`, `heart-attack?`, etc. and you probably want a way to organize these beyond having a flat list, e.g. clinical, behavioral, physiological, or other dimensions of the techniques used to determine whether the state is applicable or not and to be able to relate certain states to eachother so that e.g. after "trained" it is no longer possible to be "naive".
